packages/device_info_plus/device_info_plus/lib/src/device_info_plus_macos.dart

```diff
@@ -1 +1,2 @@
+// ignore: dangling_library_doc_comments
```

Comment:
Can you make this change on a separate PR?

That way, we have a PR per package.